runtime-modules/governance/src/council.rs

```diff
@@ -37,6 +37,20 @@ decl_storage! {
         /// because it was introduced in a runtime upgrade. It will be automatically created when
         /// a successful call to set_council_mint_capacity() is made.
         pub CouncilMint get(council_mint) : Option<<T as minting::Trait>::MintId>;
+
+        /// The reward relationships currently in place. There may not necessarily be a 1-1 correspondance with
+        /// the active council, since there are multiple ways of setting/adding/removing council members, some of which
+        /// do not involve creating a relationship.
+        pub RewardRelationships get(reward_relationships) : map T::AccountId => T::RewardRelationshipId;
```

Comment:
Why is an account id used as the identifier here? Shouldn't the relationship id just live in the representation of the council member?

Reply:
Ideally yes, the council member is represented in the `struct Seat`.
I purposefully avoided modifying the struct to avoid dealing with decoding issues after runtime upgrade. This is not the first I've had to go this sub-optimal route.
Should I refactor and introduce a custom Decode impl to deal with this migration issue?

Reply:
From the internal discussion, this is basically just due to needing to side step migration of `Seat`.

Reply:
So we ignore this issue in this release, made an issue for the future here
https://github.com/Joystream/joystream/issues/307